tests/e2e/tests/regression/insights/live-recommendations.e2e.ts

```diff
@@ -186,3 +182,18 @@ test
         // Verify that duplicates are not displayed
         await t.expect(redisVersionRecomSelector.count).eql(1, `${redisVersionRecom} recommendation duplicated`);
     });
+test.only('Verify that if user clicks on the database analysis button, new message appears  all DB', async t => {
```

Comment:
forgot to say, lets put link to the story as a comment above the test:
// https://redislabs.atlassian.net/browse/RI-4413